pages.ko/common/7z.md

```diff
@@ -0,0 +1,54 @@
+# 7z
+
+> 높은 압축률을 보여주는  파일 압축 프로그램. 
+>
+> 더 많은 정보: https://www.7-zip.org/.
+
+- 파일 또는 디렉토리 압축하기:
+
+`
+7z a {{archived.7z}} {{path/to/file_or_directory}}
+`
```

Comment:
Put the command into one line like other page.

Reply:
You mean `7z a {{archived.7z}} {{path/to/file_or_directory}}` in one line?